.github/workflows/pythonpackage.yml

```diff
@@ -205,6 +205,12 @@ jobs:
       run: |
         python -c "import vaex; df = vaex.example()"
 
+    - name: Test comand line
+      run: |
+        vaex convert ~/.vaex/data/helmi-dezeeuw-2000-FeH-v2-10percent.hdf5 test.parquet
+        pip install rich
```

Comment:
Should this be added as a proper dependency now? So we can track in case something changes with versions etc.?

Reply:
Not for now, the import is also done non-top level, if it mature, and if we are going to use rich at more places, I'm happy to make it a dependency.